libs/ledgerjs/packages/types-live/src/transaction.ts

```diff
@@ -12,10 +12,10 @@ export type SignedOperation = {
   operation: Operation;
   // usually the device signature hex OR anything that is needed to broadcast (can be an inline JSON)
   signature: string;
-  // sometimes a coin needs the raw object (it must be serializable)
-  signatureRaw?: Record<string, any>;
+  // sometimes a coin needs additional data along with the signature (it must be serializable)
+  rawData?: Record<string, unknown>;
```

Comment:
From our usage in the live we really don't know what type of value we pass in rawData ?